mentat/llm_api_handler.py

```diff
@@ -122,45 +122,39 @@ def prompt_tokens(messages: list[ChatCompletionMessageParam], model: str):
     return num_tokens
 
 
+known_models = {
+    "gpt-4-1106-preview": (128000, 0.01, 0.03),
```

Comment:
I'd prefer this tuple to be a dict with `context_size` and `pricing`. We may want to add more properties later.